controller/comments-controller.js

```diff
@@ -0,0 +1,28 @@
+const Comment = require("../models/comment");
+const Posts = require("../models/posts");
+const Scripts = require("./scripts");
+
+exports.addComment = async (req, res) => {
+  const id = req.params.postId;
+  const isValid = Scripts.isValidObjectId(id);
+  if (!isValid) {
+    return res.json({
+      err: "Invalid Id",
+    });
+  }
+
+  let newComment = new Comment({
+    desc: req.body.desc,
+    user: req.user._id,
```

Comment:
Assign all of these req content in top of function body.

Reply:
Make this change in every function.

Reply:
done